ui/src/layouts/storage.rs

```diff
@@ -242,30 +298,37 @@ pub fn FilesLayout(cx: Scope<Props>) -> Element {
                 div {
                     class: "files-breadcrumbs",
                     aria_label: "files-breadcrumbs",
-                    div {
-                        class: "crumb",
-                        aria_label: "crumb",
-                        IconElement {
-                            icon: Icon::Home,
-                        },
-                        p {
-                            "{home_text}",
-                        }
-                    },
-                    div {
-                        class: "crumb",
-                        aria_label: "crumb",
-                        p {
-                            "Folder 1"
-                        }
-                    },
-                    div {
-                        class: "crumb",
-                        aria_label: "crumb",
-                        p {
-                            "Folder 3"
+                    dirs_opened_ref.read().iter().map(|dir| {
+                        let directory = dir.clone();
+                        let dir_name = dir.name();
+                        if dir_name == ROOT_DIR_NAME {
+                            let home_text = get_local_text("uplink.home");
```

Comment:
i think this is a mistake. i can add a directory called 'root' and the breadcrumbs show Home/Home. perhaps the first directory should just be named "Home".